src/components/Rooms/AddRoom.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import '../styles/AddRoom.css';
+
+function AddRoom() {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [image, setImage] = useState('');
+  const [size, setSize] = useState('');
+  const [view, setView] = useState('');
+  const [bedding, setBedding] = useState('');
+  const [message, setMessage] = useState('');
+
+  const submitRoom = (e) => {
+    e.preventDefault();
+    const room = {
+      name,
+      description,
+      image,
+      size,
+      view,
+      bedding,
+    };
+
+    function notify(response) {
```

Comment:
- [ ] Kindly refactor this function to use ES6.   Reason:  The vast majority of the software industry has deprecated the`function` keyword.

That way, your app will comply with the latest standards 💪

Reply:
Thank you so much for the feedback, Miguel. I will handle this issue.